Resources/conandeps.props

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" ?>
```

Comment:
what is this file for?

Reply:
this is an empty conandeps.props to be able to inject the properties before the props are even generated, but I can't get it to work right the first time the project is built, we can have a discussion about what would be the best approach....

Reply:
I finally removed it